store/sqlstore/user_access_token_store.go

```diff
@@ -182,35 +188,36 @@ func (s SqlUserAccessTokenStore) GetByUser(userId string, offset, limit int) ([]
 func (s SqlUserAccessTokenStore) Search(term string) ([]*model.UserAccessToken, error) {
 	term = sanitizeSearchTerm(term, "\\")
 	tokens := []*model.UserAccessToken{}
-	params := map[string]interface{}{"Term": term + "%"}
+	var params []interface{}
+	params = append(params, term, term, term)
```

Comment:
Is there any difference with doing this in one line like below?
```suggestion
	params := []interface{}{term, term, term}
```

Reply:
Hi @isacikgoz! I followed the approach of first declaring and then appending elements to the params slice as I had followed a similar approach in one of the previous PRs I had created, ref. - https://github.com/mattermost/mattermost-server/pull/18633/files#diff-ad4af66ae20aebf82d0052b56640f5a65d432ef684d65c9cdb822dc5830618a1R167-R176
Hence, to have a consistent style I implemented it in such a manner.

Reply:
Hey @penthaapatel - in the earlier scenario, we didn't know what items to append because they were dependent on the if condition. In this case, I think it should be fine to write the slice with all the elements pre-added. Not that it's a big deal either ways, just a matter of preference.

Reply:
@agnivade Thank you for clearing that up. Shoud I push the suggested change or is it okay the way it is currently?

Reply:
You can push. We can merge after that. Thanks :)

Reply:
Done! :)